src/components/action-popover/action-popover.stories.mdx

```diff
@@ -230,7 +230,7 @@ used actions, in this example "manage devices".
 It is possible to utilise the selected and onClick props on the FlatTableRow to highlight single rows of data.
 
 <Canvas>
-  <Story name="in FlatTable" parameters={{ chromatic: { disable: true } }} story={stories.ActionPopoverComponentInFlatTable} />
+  <Story name="in FlatTable" story={stories.ActionPopoverComponentInFlatTable} />
```

Comment:
Why is the disabled Chromatic no longer required here? Is it because it's disabled for all at line 17 in the Meta? Can none of the Action Popover stories be enabled for Chromatic to snapshot?

Reply:
yes